webroot/adm/oncall/search.html

```diff
@@ -69,6 +68,8 @@ <h5>Search Results:</h5>
             }
         });
     </script>
+    <script src="../../js/main.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-YvpcrYf0tY3lHB60NNkmXc5s9fDVZLESaAA55NDzOxhy9GkcIdslK1eN7N6jIeHz" crossorigin="anonymous"></script>
```

Comment:
My understanding is that putting the script tags here at the end of the body are better for performance. Should we consider doing this in the other files as well that use bootstrap as well? i.e. `generate-api-key-secret.html` and `generate-cstg-keypair.html`?

Reply:
it looks like those scripts are already being loaded at the bottom of the body.

Reply:
ah true. I think I got mixed up with the `main.js` script tag being in the `<head>` instead in a bunch of other files (e.g. [client-key.html](https://github.com/IABTechLab/uid2-admin/blob/main/webroot/adm/client-key.html#L4). Probably not worth changing though.